bin/whisper-diff.py

```diff
@@ -63,9 +66,25 @@ def print_summary(diffs,pretty=True,headers=True):
     sys.stdout.write(f%('archive','total','differing'))
   for archive, points, total in diffs:
     sys.stdout.write(f%(archive,total,points.__len__()))
-    
-archive_diffs = whisper.diff(path_a,path_b,ignore_empty=options.ignore_empty,until_time=until_time)
-if options.summary:
-  print_summary(archive_diffs,pretty=(not options.columns),headers=(not options.no_headers))
-else:
-  print_diffs(archive_diffs,pretty=(not options.columns),headers=(not options.no_headers))
\ No newline at end of file
+
+def print_summary_json(diffs,path_a,path_b):
+  print json.dumps({'path_a': path_a, 'path_b': path_b, 'archives': [{'archive': archive, 'total': total, 'points': points.__len__()} for archive, points, total in diffs]}, sort_keys=True, indent=2, separators=(',', ' : '))
```

Comment:
These lines are way too long. We're not going to be overly pedantic about PEP conventions here, but we should try to limit them to a reasonable length. See https://www.python.org/dev/peps/pep-0008/#maximum-line-length for some guidelines.